### PR TITLE
chore(tools): remove devdependency lodash-es; used once, ~622kB

### PR DESCRIPTION
**Pull request by iambumblehead**
Removes lodash-es devdependency, ~622kB
 * https://packagephobia.com/result?p=lodash-es

### CODE REVIEW

File: test/unit/core.ts [hunk 1]
```diff
@@ -848,7 +847,7 @@ describe("snabbdom", function () {
               return spanNumWithOpacity(n, "1");
             }),
           );
-          const shufArr = shuffle(arr.slice(0));
+          const shufArr = arr.slice(0).sort(() => 0.5 - Math.random());
```
Comment (paldepind) [inline test/unit/core.ts:850]: This "[Microsoft shuffle](https://www.robweir.com/blog/2010/02/microsoft-random-browser-ballot.html)" is not a good way of shuffling an array. See also [the comments on this SO answer](https://stackoverflow.com/a/18650169/556072).

If we want to remove the Lodash dependency I think we should define our own `shuffle` function that does a proper shuffle. The implementation in the bottom of [this SO answer](https://stackoverflow.com/a/12646864/556072) seems like a fine candidate.

Comment (iambumblehead) [reply test/unit/core.ts]: the SO shuffle is added here, but maybe lodash-shuffle was better than using a custom function.

feel free to close this PR or merge either way

File: test/unit/core.ts [hunk 2]
```diff
@@ -31,6 +30,17 @@ const patch = init(
   { experimental: { fragments: true } },
 );
```
Comment (paldepind) [inline test/unit/core.ts:32]:
```suggestion
/** Shuffle an array using Durstenfeld's version of the Fisher–Yates shuffle. */
```

Comment (iambumblehead) [reply test/unit/core.ts:32]: this was added locally, rather than through the github interface to preserve the git commit message formatting

**Review by paldepind — APPROVED** [review]
Looks good! 🎉

**Pull request merged**